### PR TITLE
grey box for 4 level nav

### PR DESCRIPTION
**Pull request by tinglisD2L**
Acceptance Criteria met:
- Add support for a new third level of depth within a sequence.
- This third level is "flattened" int he sense that there is not drill down or animation to expand it.
- Third level is indented slight from the second level.
- Third level has a grey background
- Third level does not track completion of items within that level
- Completion of items within the third level are tracked / included in the completion tracking for the module above it (e.g. "How to develop and teach..." includes completion numbers for its child module "Supporting leadership with diversity")
- Iterating through the sequence will take you into the third level
- When on the last topic in the third level module, iterating next will take you to the topic in the parent module that follows the third level module
- Mouse over effects for the third level are the same as other levels (light blue)

AC not met:
- Iterating through the sequence will take you into the third level, starting with the module description page first. 
- Mouse over effects for the third level are the same as other levels (light shade of course primary colour)
- When selecting the third level module / topic, that item is loaded in the viewer, and is coloured with the course primary colour (e.g. If student clicked on "Video: Diversity in the Office" it it would be coloured the green colour, and Personal Adaptabilty would go back to be coloured white.

### CODE REVIEW

File: components/d2l-module-link.html [hunk 1]
```diff
@@ -29,103 +46,98 @@
...
-
-			.divider-icon {
-				color: var(--d2l-color-mica) !important;
-			}
-
 		</style>
 
 		<d2l-accordion-collapse
-			class$="[[_getIsSelected(selectedModule)]]"
+			class$="[[_getIsSelected(selectedModule)]] [[_getIsInnerModule(innerModule)]]"
```
Comment (jwong-d2l) [inline components/d2l-module-link.html]: I think part of the reason why you have so many test failures is that you've have two components that are trying to do different things combined into one component.  The module link component shouldn't be the same component as the module header component, I don't think.

**Comment by tinglisD2L** [comment]
@jwong-d2l What do you mean module-link is sometimes not a link? I just made the accordion so that it's sometimes not collapsible. Making module-link a link (as in, showing the module description in the viewer) is another task in the story.

**Comment by jwong-d2l** [comment]
Okay, maybe I'm just not really following what's going on because it's so busy. What I was trying to say was that in my mind, the module link component is a clickable item, and by having it sometimes always be open, you're kind of representing a different kind of component that doesn't belong in that class, although it uses the same type of entity.

**Comment by jwong-d2l** [comment]
I misunderstood the acceptance criteria, and I've gotten Josh to help me update it.  This looks good to go!

**Pull request merged**